lib/node_modules/@stdlib/stats/base/nanmin/lib/nanmin.js

```diff
@@ -41,46 +41,8 @@ var isNegativeZero = require( '@stdlib/math/base/assert/is-negative-zero' );
 * var v = nanmin( N, x, 1 );
```

Comment:
Let's go ahead and remove L39 and replace `N` here with `x.length`. The use of `N` in this example doesn't really add anything.